micro-rdk/src/common/robot.rs

```diff
@@ -265,6 +274,11 @@ impl LocalRobot {
             data_manager_collection_task: None,
             start_time: Instant::now(),
         };
+        if let Some(cloud_cfg) = config.cloud.as_ref() {
```

Comment:
Is there some nice method on `Option` you could use above on 263-265 to achieve this in one go? An `or_default` type thing?

Reply:
there's something I can do that is less efficient, I'll push it up and see if you prefer it